cmake/macros/SundialsAddTest.cmake

```diff
@@ -92,6 +98,11 @@ macro(SUNDIALS_ADD_TEST NAME EXECUTABLE)
 
       if(SUNDIALS_TEST_PROFILE)
         list(APPEND TEST_ARGS "--profile")
+        if (SUNDIALS_CALIPER_OUTPUT_DIR)
+          list(APPEND TEST_ARGS "--calidir=${SUNDIALS_CALIPER_OUTPUT_DIR}/Testing/${JOB_ID}")
```

Comment:
Is `/Testing/` needed still?

Reply:
It's kept there to organize the Caliper files by benchmarks and test suite runs.